django-hatstall/hatstall/views.py

```diff
@@ -189,8 +189,8 @@ def merge_profiles(request):
     """
 
     if request.method == 'POST':
-        err = merge(request.POST.getlist('uuid'))
-    return redirect('identities list')
+        last_uuid, err = merge(request.POST.getlist('uuid'))
+    return redirect('/identities/hatstall/' + last_uuid)
```

Comment:
There should be a better way to do this. This depends on how you configure and set paths in `urls.py`. If those values change, this won't work.

Reply:
You are right! I'm going to change it to the "url" name defined. Sorry!